RELEASING.md

```diff
@@ -0,0 +1,13 @@
+# Releasing
+
+Outline for releasing this package to Hackage and Stackage
+
+## Steps
+
+* `git tag <version> -m "Release version <version>"`
+* `git push origin --tags`
+* `stack sdist`
+* Copy to a folder that can be navigated to for upload
+* Upload here: http://hackage.haskell.org/packages/upload
+* `scripts/hackage-docs.sh <user>`
```

Comment:
What's the point of using hackage-docs.sh. Isn't the hackage automatic doc generation sufficient for our needs?

Reply:
I haven't ever gotten that to work. This is what I ran to get the docs up in their current state, this stuff is just fire and forget for me.

Reply:
Cool. As long as this works, I have no objections.